source/version_helper.js

```diff
@@ -10,5 +11,34 @@ module.exports = {
       // in that case we fallback to a branch version
       return semver.coerce(version === '0.0.0-development' ? branchVersion : version);
   },
+  CockroachVersionGreaterThan21_1: async function(connection) {
```

Comment:
"greater than 21.1" sounds more like it means strictly greater. maybe the function can be named `IsCockroach21_1Plus`